ppr-ui/src/resources/statusTypes.ts

```diff
@@ -38,3 +37,17 @@ export const MhStatusTypes = [
     text: MhUIStatusTypes.HISTORICAL
   }
 ]
+
+export const PprAPIToUIStatusTypesMap = {
+  [APIStatusTypes.DRAFT]: UIStatusTypes.DRAFT,
+  [APIStatusTypes.ACTIVE]: UIStatusTypes.ACTIVE,
+  [APIStatusTypes.EXPIRED]: UIStatusTypes.EXPIRED,
+  [APIStatusTypes.DISCHARGED]: UIStatusTypes.DISCHARGED
+}
+
+export const MhrAPIToUIStatusTypesMap = {
+  [MhApiStatusTypes.DRAFT]: MhUIStatusTypes.DRAFT,
+  [MhApiStatusTypes.ACTIVE]: MhUIStatusTypes.ACTIVE,
+  [MhApiStatusTypes.EXEMPT]: MhUIStatusTypes.EXEMPT,
+  [MhApiStatusTypes.HISTORICAL]: MhUIStatusTypes.HISTORICAL
+}
```

Comment:
I like this approach! 
No action required but have a look if the above value/text objects are something we still* need.

Reply:
I believe the above array makes it simple to use vue model in rendering the select, since it specifies the value and text properties for the select, that's why I did not get rid of it.

![image](https://github.com/bcgov/ppr/assets/77707952/d3667776-b270-4536-8bd6-2f7a02f9b36e)

Reply:
Excellent, as long as you gave it consideration, which you clearly have :)